src/backend/db/categories.js

```diff
@@ -0,0 +1,27 @@
+import { v4 as uuid } from "uuid";
+
+/**
+ * Category Database can be added here.
+ * You can add category of your wish with different attributes
+ * */
+
+export const categories = [
+  {
+    _id: uuid(),
```

Comment:
i suggest use any random id instead of uuid bcz for now its fine but when u work on single product page will give error bcz on each render uuid changes but u have to fetch data from same id